client/me/account-password/index.jsx

```diff
@@ -90,7 +92,7 @@ module.exports = React.createClass( {
 					debug( 'Error saving password: ' + JSON.stringify( error ) );
 
 					// handle error case here
-					notices.error( this.translate( 'There was a problem saving your password. Please, try again.' ) );
+					this.props.errorNotice( this.translate( 'There was a problem saving your password. Please, try again.' ) );
```

Comment:
since this line runs on a bit, why not go ahead and destructure these props?

```js
submitForm: function( event ) {
	const { userSettings: { saveSettings }, errorNotice } = this.props;

	…

	saveSettings(
		…
		errorNotice( this.translate( 'There was a problem saving your password. Please, try again.' ) );
		…
	);
}
```